compare.py

```diff
@@ -5,11 +5,12 @@
 Result = namedtuple("Result", ["name", "base_time", "diff_time"])
 
 def get_times(pytest_data):
-    return {b["name"]: b["stats"]["mean"] for b in pytest_data["benchmarks"]}
+    return {b["name"]: b["stats"]["min"] for b in pytest_data["benchmarks"]}
```

Comment:
I always feel weird about using `min` for benchmarking, but it has its advantages, so I'm cool with it.

Reply:
Oh, I didn't notice I changed that, will undo to avoid creeping in undesired changes.